mule/sign.sh

```diff
@@ -21,7 +21,7 @@ make_hashes () {
     for ext in ${EXTENSIONS[*]}
     do
         if ls *"$VERSION"*."$ext"; then
-            HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext}"
+            HASHFILE="hashes_${OS_TYPE}_${ARCH}_${VERSION}_${ext/./}"
```

Comment:
@onetechnical I followed the description on my ticket, but is this correct?

Reply:
It looks like this removes the period from `tar.bz2`? Seems fine to me if that's the intent.